salt/modules/smtp.py

```diff
@@ -76,6 +82,7 @@ def send_msg(recipient,
              use_ssl='True',
              username=None,
              password=None,
+             attachments=None,
```

Comment:
Can you move this to the end of the list so we don't cause an API error?

Reply:
Ah, of course! Moved to the end, thanks for spotting it. :)